BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
+
+function generateColors() {
+  const colors = {};
+  const randomColor = () => {
+    const r = Math.floor(Math.random() * 256);
+    const g = Math.floor(Math.random() * 256);
+    const b = Math.floor(Math.random() * 256);
+    return `rgb(${r},${g},${b})`;
+  };
+
+  for (let i = 0; i < TIME; i++) {
```

Comment:
Wouldnt this cause problems if the product specs changes?  For example, what happen if we change the spec to  change the background color to every .5 seconds for 10.7 seconds?